cfpb/consumerfinance.gov
# Break apart our single global Spanish translation file

This changes breaks apart the large, global Spanish translation files in `cfgov/locale/` so that messages that come from individual apps (like `ask_cfpb`, `v1`, etc) are stored in their apps.

This includes all the messages that come from our templates after https://github.com/cfpb/consumerfinance.gov/pull/7402, which relocated the majority of them from `cfgov/jinja2/v1` into their respective app packages.

## How to test this PR

After checking out and running the site with this PR, ensure that translations within each relevant app work.

For example, the search controls in Ask CFPB: http://localhost:8000/es/obtener-respuestas/buscar/?q=

The Financial Well-being tool http://localhost:8000/consumer-tools/financial-well-being/es/

## Checklist
